apko/private/apk.bzl

```diff
@@ -15,6 +15,29 @@ filegroup(
 )
 """
 
+def _auth(rctx, url):
+    http_auth = rctx.getenv("HTTP_AUTH")
```

Comment:
It looks like this only works in more recent versions of Bazel.

@thesayyn @alexeagle what's the best way of doing this in a portable way?

Reply:
There is history on rules_oci for this: we carried a fallback of spawning 'curl' for this